masterlist.yaml

```
@@ -8043,6 +8043,20 @@ plugins:
         itm: 24
         udr: 21
 
+  - name: 'XSPipeMod.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/11892/'
+        name: 'Smokeable Pipe Mod'
+    msg:
+      - <<: *versionXofYorGreaterRequired
+        subs:
+          - '0.0.10.0'
+          - '[OBSE](https://obse.silverlock.org) v10+'
```

The version number displayed should match what's reported on OBSE's website as opposed to how it's recorded in the file's header. The `v10+` is also redundant with this anchor.

```suggestion
          - '10'
          - '[OBSE](https://obse.silverlock.org)'
```